modules/nf-core/cowpy/tests/main.nf.test

```diff
@@ -0,0 +1,122 @@
+nextflow_process {
+
+    name "Test Process COWPY"
+    script "../main.nf"
+    process "COWPY"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "cowpy"
+
+    config "./nextflow.config"
+
+    test("hello") {
+
+        when {
+            process {
+                """
+                def text = "hello"
+                def text_file = new File('hello.txt')
+                text_file.text = text
```

Comment:
FYI this could probably be simpler with collectFile, but it's fine.

Reply:
yeah, I badly copy pasted from another test